stl/inc/tuple

```diff
@@ -570,6 +692,47 @@ public:
         _Get_rest()._Myfirst._Val = _STD forward<_Second>(_Right.second);
         return *this;
     }
+
+#ifdef __cpp_lib_concepts
+    template <bool _Const_assignment, class _Other, class _Indices = index_sequence_for<_Rest...>>
+    static constexpr bool _Can_assign_values_from_tuple_like_v = false;
+
+    template <bool _Const_assignment, _Tuple_like _Other, size_t... _Indices>
+    static constexpr bool _Can_assign_values_from_tuple_like_v<_Const_assignment, _Other, index_sequence<_Indices...>> =

```

Comment:
Done!

> I don't love the conditional_ts; it seems a little bit too clever, to me

It could be worst, I thought about using `_Maybe_const<_Const_assignment, _This>&` here 🙀